src/CMakeLists.txt

```diff
@@ -22,23 +22,29 @@ set_target_properties           (leptonica PROPERTIES OUTPUT_NAME       leptonic
 set_target_properties           (leptonica PROPERTIES DEBUG_OUTPUT_NAME leptonica-${VERSION_PLAIN}d)
 endif()
 
-if (NOT STATIC)
+if (BUILD_SHARED_LIBS)
     target_compile_definitions  (leptonica PRIVATE -DLIBLEPT_EXPORTS)
 endif()
 
+target_include_directories(
+    leptonica
+    INTERFACE
+    "$<INSTALL_INTERFACE:include/leptonica>"
```

Comment:
> I had ignored this but Tesseract does #include <allheaders.h>

May be it means we need to fix Tesseract? Because otherwise the `mystuff/a.hpp` issue will propagate to **all** targets that use leptonica (or have leptonica as dependency).

> Does INCLUDES DESTINATION allow multiple destinations?

I don't know actually. Well if we need `include/leptonica` anyway then it's okay to keep it like that.